src/components/visualizations/utils/drilldownEventing.ts

```diff
@@ -472,3 +473,29 @@ export const getDrillIntersection = (
         [],
     );
 };
+
+export function handleGeoPushpinDrillEvent(
+    drillConfig: IDrillConfig,
+    drillableProps: IDrillableProps,
+    _target: EventTarget,
+): void {
+    const { afm, onDrill } = drillConfig;
```

Comment:
no support for legacy onFiredDrillEvent? So we can have it only optional in interface?

Reply:
and I would mention it in documentation, this is first chart where providing onFiredDrillEvent handler prop does nothing

Reply:
I have added onFiredDrillEvent